app/views/application/personal-details/residency.njk

```diff
@@ -11,8 +11,10 @@
 
 {% block primary %}
 
-<p class="govuk-body">To get help with student visas and your immigration status, <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">register with Get Into Teaching</a> . You will be able to speak to an adviser.</p>
-<p class="govuk-body">Most School Centred Initial Teacher Training (SCITT) providers cannot sponsor student visas for trainees on unsalaried courses.</p>
+<p class="govuk-body">To get help with student visas and your immigration status,speak to a <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">Get Into Teaching</a> adviser.</p>
```

Comment:
```suggestion
<p class="govuk-body">To get help with student visas and your immigration status, speak to a <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">Get Into Teaching</a> adviser.</p>
```

@Tomcowan1 Should we be linking to the adviser service here? https://beta-adviser-getintoteaching.education.gov.uk